esp-hal/src/dma/mod.rs

```diff
@@ -1534,25 +1522,40 @@ pub trait DmaChannel: crate::private::Sealed {
 
     /// A description of the TX half of a DMA Channel.
     type Tx: TxRegisterAccess + InterruptAccess<DmaTxInterrupt>;
+}
 
+/// A description of a DMA Channel that can be used with a peripheral.
+#[cfg(pdma)]
+pub trait PeripheralDmaChannel: DmaChannel {
```

Comment:
Looks like this trait isn't needed anymore. At least I haven't spotted it being consumed.

Reply:
Possibly, I'll clean this up with a fresh mind.